packages/react/src/components/Dialog/Dialog.props.ts

```diff
@@ -0,0 +1,60 @@
+import { PortalProps } from '../Portal'
+
+export type DialogSizes = 'small' | 'medium' | 'large'
+
+export interface DialogProps {
+  /**
+   * The id(s) of the element(s) that describe the dialog.
+   */
+  ariaDescribedBy?: string
+
+  /**
+   * The id(s) of the element(s) that label the dialog.
+   */
+  ariaLabelledBy?: string
+
+  /**
+   * Dialog children, usually the included sub-components.
+   */
+  children: React.ReactNode
+
+  /**
+   * An HTML element or function that returns one. This element will be used
+   * as the container for the Dialog and can be used to control where will be rendered
```

Comment:
acho que faltou um 'it', tipo: _...and can be used to control where **it** will be rendered_